lib/remap-lang/grammar.pest

```diff
@@ -96,6 +96,32 @@ regex_char  =  { !("/" | "\\") ~ ANY | "\\" ~ ANY }
 
 kv_pair = { string ~ ":" ~ expression }
 
+reserved_keyword = @{
+    "if"
+    | "else"
+    | "for"
+    | "while"
+    | "loop"
+    | "abort"
+    | "break"
+    | "continue"
+    | "return"
+    | "as"
+    | "type"
+    | "let"
+    | "until"
+    | "then"
+    | "match"
+    | "impl"
+    | "in"
+    | "self"
+    | "this"
+    | "use"
+    | "std"
+    | null
+    | boolean
```

Comment:
Not a big deal but I was surprised to see `boolean` reserved here but not `string` or any other type names.

I might add a comment indicating that not all of these are actually used right now to avoid confusion for people working in remap-lang.

Reply:
Ah. To be clear: these last two are _rules_ (see the missing quotes) whereas the previous ones are string literals.

That is, this `boolean` rule expands to two string literals: `"true"` and `"false"`, similar to how the `null` rule expands to the `"null"` string literal.

> I might add a comment indicating that not all of these are actually used right now to avoid confusion for people working in remap-lang.

Good call.

Reply:
Aha! I see. :+1: